tests/periph_eeprom/tests/01-run.py

```diff
@@ -11,9 +11,21 @@
 
 
 def testfunc(child):
+    # make sure the shell is ready before sending the test command
+    child.expect_exact('>')
     child.sendline('test')
     child.expect('SUCCESS')
 
+    # reboot the device to ensure data are still available on EEPROM
+    child.sendline('reboot')
```

Comment:
This will require blacklisting boards using `stdio_cdc_acm`

Reply:
Indeed, but `stdio_cdc_acm` is not a feature that could be blacklisted. I could at minimum blacklist `bootloader_arduino` or `bootloader_nrfutil` but that doesn't cover all boards using `stdio_cdc_acm`.
For the moment, the good thing is (I think) that there's no board providing `periph_eeprom` and using `stdio_cdc_acm`.

Reply:
Lets leave it like that for now, this is not the only test with that issue.